src/components/app/App.less

```diff
@@ -23,6 +23,7 @@
     width: 100%;
     padding-left: 75px;
     background-color: #3E4347 !important;
+    z-index: 10000 !important;
```

Comment:
I actually think I just found a solution, but I'm going to work on it in my "UI fixes" branch once this is merged in, because the solution involves using a theme (which I was already doing for typography on that branch anyways).